device/mellanox/x86_64-mlnx_msn2010-r0/sensors.conf

```diff
@@ -47,3 +47,25 @@ bus "i2c-2" "i2c-1-mux (chan_id 1)"
         label fan2 "Chassis Fan 2"
         label fan3 "Chassis Fan 3"
         label fan4 "Chassis Fan 4"
+        ignore temp2
```

Comment:
I remember there are 16 ports in 2010, but there are 20 ignore here

Reply:
2010 has 22 ports, 18 25G + 4 100G, so we need to ignore 22 sensors here